backend/app/api/api_v1/endpoints/webhook.py

```
@@ -0,0 +1,173 @@
+    return results
+
+
+def _subject_from_message(msg: email.message.Message, fallback: Optional[str] = None) -> str:
+    return fallback or _decode_email_header(msg.get("Subject"))
+
+
+@router.post("/email")
+async def receive_email(
+    payload: EmailWebhookPayload,
+    x_webhook_secret: Optional[str] = Header(None),
+    db: Session = Depends(get_db),
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)